tingbot/error.py

```diff
@@ -43,11 +43,7 @@ def sad_tingbot_image():
 
 
 def error_screen(exc_info):
-    surface = pygame.display.get_surface()
-    if not surface:
-        return
-
-    screen = graphics.Surface(surface)
+    screen = graphics.screen
 
```

Comment:
Okay... this might not be the right approach as I'd rather not make the `screen` object if the user is using the run loop but not using the `graphics` module from tingbot-python, for example using pygame directly, or not using the screen at all.

I've added a has_surface property to Screen on master, could this be used instead?